charts/reform-scan-blob-router/requirements.yaml

```diff
@@ -12,3 +12,8 @@ dependencies:
     version: ~0.2.1
     repository: '@hmctspublic'
     condition: cftblobstorage.enabled
+  - name: blobstorage
+    alias: crimeblobstorage
+    version: ~0.2.1
+    repository: '@hmctspublic'
+    condition: cftblobstorage.enabled
```

Comment:
We have three blob storage charts in dependencies - reform, cft and crime. Do we need all three? Also, should the condition be different for crime?

Reply:
we want to create 3 different blob storage so we can test moving from one blob storage to another one, for helm alias is only way for now

Reply:
I got the three blob storage charts now, just the question about the condition